### ISSUE TITLE
Get chart not working

### ISSUE THREAD

**Issue by 1616234** [93Alliance/ng-chartjs#4]
I tried using the method described in the main page to get the chart istance, but it isn't working. The getChart method returns null despite the id being the correct one.
Because of this, I can't call chart.update(). I've been able to have the chart update data/labels automatically through replacing arrays, but this doesn't work for options, so I don't know how to change them after creating the chart (assigning a new options object does nothing). 
I also see there is a resetOption field, but it's not clear how to use it and my understanding is it can't be called from the javascript side, only the template (which doesn't do it for me, I need to do it before putting new data into the chart without input from user).
What i want to do is change the label for my y axis after the chart has been created. Is there any fix or workaround for this?

**Comment by 93Alliance** [comment]
Can you show me your demo?

**Comment by 93Alliance** [comment]
example

`html`
```
<canvas id="testChart" ngChartjs [datasets]="lineChartData" [labels]="lineChartLabels"
      [options]="lineChartOptions" [legend]="lineChartLegend" [chartType]="lineChartType" ></canvas>
```
`ts`

First you need to import the service.
```
import { NgChartjsService } from 'ng-chartjs';
```
Call the `getChart` function to get the `chart.js` instance.
```
 const chart: any = this.ngChartjsService.getChart('testChart');
 chart.options.scales.xAxes[0].ticks.display = false;
 chart.update();
```

**Comment by 1616234** [comment]
I've figured out why it isn't working: while everything was exactly the same as your code, I was using getChart inside the NgOnInit() method. I thought you were doing the same since your main page example shows the code inside NgInit(). I'm not sure which angular version is NgInit() from, but in Angular 7 NgOnInit() is evidently executed before the chart is fully created, therefore I was only getting null back. I realised this by moving the method to a ChartClicked event, and am now receiving an object correctly. I assume I should therefore use NgAfterViewInit() instead to have the desired effect.
Thank you for the swift response in any case.
